pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/transformers/MarcSubjects.scala

```diff
@@ -0,0 +1,32 @@
+package weco.pipeline.transformer.marc_common.transformers
+import weco.catalogue.internal_model.identifiers.IdState
+import weco.catalogue.internal_model.work.Subject
+import weco.pipeline.transformer.marc_common.models.MarcRecord
+import weco.pipeline.transformer.marc_common.transformers.subjects.{
+  MarcConceptSubject,
+  MarcMeetingSubject,
+  MarcOrganisationSubject,
+  MarcPersonSubject,
+  MarcSubject
+}
+
+object MarcSubjects extends MarcDataTransformer {
```

Comment:
This behaviour differs from Sierra, which does something weird WRT order of subjects - see [Slack](https://wellcome.slack.com/archives/C02ANCYL90E/p1713877204246529)